examples.json

```diff
@@ -147,6 +147,12 @@
         "keywords": "datepicker"
       },
       {
+        "title": "UI Timepicker",
+        "slug": "ui-timepicker",
+        "jsbinId": "palesa",
+        "keywords": "timepicker"
```

Comment:
Note, this is technically not necessary because the title and slug both have the word `timepicker` in it. But no worries, we'll leave it in there :-)